### PR TITLE
Rating toast notify snackbar

### PR DESCRIPTION
**Pull request by prakash855**
Rating for e-com & beautiful toasts integrated in this application now!

### CODE REVIEW

File: components/Button/Button.css [hunk 1]
```diff
@@ -30,6 +33,7 @@
 }
```
Comment (AtulPant2704) [inline components/Button/Button.css:34]: Since you are using same transition multiple times, you can assign a variable to it.

**Review by AtulPant2704 — COMMENTED** [review]
rating

Your rating component size is very small, you should increase it.

**Review by Tejinder15 — COMMENTED** [review]
Suggestions: Try Making Pages Responsive , You can change hyperlink of getting started to # in href instead of giving index page.
Also on modal page correct the footer navigation.

File: components/Button/Button.css (hunk 1 above)
Comment (prakash855) [reply components/Button/Button.css:34]: Sure thanks

**Pull request merged**